app/main/views/services.py

```diff
@@ -200,18 +207,14 @@ def import_service(service_id):
 
     db.session.add(service)
 
-    try:
-        db.session.commit()
-    except IntegrityError:
-        db.session.rollback()
-        abort(400, "Unknown supplier ID provided")
+    db.session.commit()
+    search_api_client.index(service_id, service.data, supplier.name)
```

Comment:
Is this right? Don't we still want to return a 400 if the supplier ID is unknown?